modules/sfp_bitcoinwhoswho.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_bitcoinwhoswho
+# Purpose:      Bitcoin Who's Who database lookup module
+#
+# Author:      Leo Trubach <leotrubach@gmail.com>
+#
+# Created:     2020-09-09
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+import urllib.parse
+
+from spiderfoot import SpiderFootEvent, SpiderFootPlugin
+
+
+class sfp_bitcoinwhoswho(SpiderFootPlugin):
+    __name__ = 'sfp_bitcoinwhoswho'
+    meta = {
+        'name': "Bitcoin Who's Who",
+        'summary': "Check for malicious Bitcoin address agains BitcoinWhosWho database",
```

Comment:
Typo (agains -> against) and we should be consistent with the name - is it BitcoinWhosWho or Bitcoin Who's Who?